### PR TITLE
Updating README to have more background, example usage, and more detailed installation instructions

### PR DESCRIPTION
**Pull request by maxlindbergh**
# Summary

I've added some project background to make the goals and capabilities of this project more clear to people not familiar with the codebase. Additionally, I've added an "Example Usage" section to the README so that it is, again, more clear to new users how they can use this software.

Finally, I've added a bit of description to each of the three installation options. I've put in some information on how they differ and when each case should be used.

# Test plan

This is purely a documentation change so no real testing is required.

### CODE REVIEW

**Review by hoxyq — APPROVED** [review]
Thanks.

File: README.md [hunk 1]
```diff
@@ -2,8 +2,22 @@
...
+## Example usage
+After building the project, you can use the React Native Debugger frontend to inspect and debug your React Native app. First, serve the built files 
+locally using:
+
+```sh
+python3 -m http.server 8000 --directory out/Default/gen/front_end
+```
+Then, open your browser and navigate to http://localhost:8000/inspector.html for general debugging, or http://localhost:8000/rn_inspector.html for 
+React Native-specific debugging. This allows you to inspect UI components, monitor network requests, and analyze performance metrics directly in 
+your browser using Chrome DevTools.
```
Comment (hoxyq) [inline README.md:19]:
```suggestion
Then, open your browser and navigate to http://localhost:8000/rn_fusebox.html for 
React Native-specific debugging.
```

**Review by hoxyq — CHANGES_REQUESTED** [review]
Sorry, missed this one. We only use `rn_fusebox` as a custom entrypoint for React Native applications.

File: README.md [hunk 2]
```diff
@@ -21,18 +35,28 @@ This repository is a fork of [ChromeDevTools/devtools-frontend](https://github.c
...
+
     ```sh
     npm run build
     ```
 
 1. Build with the release config once:
 
+   This one-time build optimizes the code for production use. It’s slower to compile and can take significantly more time but results in a stable,
+   high-performance version suitable for production environments. Use this when preparing the project for final deployment or release.
+
```
Comment (huntie) [inline README.md:59]: These bits feel too verbose (AI generated?) — these scripts are self-documenting. Intend to come back and merge this PR, but let's do a pass of the file which removes these.

**Pull request closed without merging**